### PR TITLE
Translations for 20.21.02

### PR DESCRIPTION
**Pull request by bemailloux**
Testing: did a quick check of the main page and user drill to make sure everything should appear is in the correct language. 

I didn't check this string: `userDrill:manyCoursesAlert`.

There is an issue with the pagination control in zh (PRC) but that's a pagination repo problem:
image

Fix for the above here: https://github.com/BrightspaceUILabs/pagination/pull/27

### CODE REVIEW

File: locales/cy-gb.js [hunk 1]
```diff
@@ -209,10 +212,30 @@ export default {
...
+
 	"userOverdueAssignmentsCard:assignmentsCurrentlyOverdue": "mae aseiniadau’n hwyr ar hyn o bryd.",
-	"averageGradeSummaryCard:averageGradeText" : "grade averaged from the courses in view.",
+	"averageGradeSummaryCard:averageGradeText" : "gradd ar gyfartaledd o’r cyrsiau mewn gwedd.",
 	"averageGradeSummaryCard:averageGrade" : "Gradd ar Gyfartaledd",
-	"averageGradeSummaryCard:noGradeInfoAvailable" : "No grade information available.",
-	"userSysAccessCard:daysSinceLearnerHasLastAccessedSystem": "days since the learner last accessed the system.",
-	"userSysAccessCard:userHasNeverAccessedSystem": "User has never accessed the system."
+	"averageGradeSummaryCard:noGradeInfoAvailable" : "Nid oes unrhyw wybodaeth gradd ar gael.",
+	"userSysAccessCard:daysSinceLearnerHasLastAccessedSystem": "diwrnodau ers i’r dysgwr gyrchu'r system ddiwethaf.",
```
Comment (johngwilkinson) [inline locales/cy-gb.js:223]: a couple of inconsistent apostrophes in this term and the next - maybe on purpose, but it seems unlikely

**Review by johngwilkinson — APPROVED** [review]

File: locales/cy-gb.js (hunk 1 above)
Comment (johngwilkinson) [reply locales/cy-gb.js:223]: linter doesn't like something here

Comment (bemailloux) [reply locales/cy-gb.js:223]: Enforcing double quotes to denote "this is a string" in the Welsh file seems to be the easiest option here

**Review by anhill-D2L — APPROVED** [review]

**Comment by bemailloux** [comment]
Oof, almost merged into master 😅

**Comment by johngwilkinson** [comment]
> Oof, almost merged into master 😅

We need it in master too

**Comment by bemailloux** [comment]
> > Oof, almost merged into master 😅
> 
> We need it in master too

Yup, #375 still needs approval

**Pull request merged**